objectbox/lib/src/native/query/query.dart

```diff
@@ -28,7 +28,8 @@ class Order {
   /// If not specified, the default is case insensitive for ASCII characters.
   static final caseSensitive = 2;
 
-  /// For scalars only: changes the comparison to unsigned (default is signed).
+  /// For integers only: changes the comparison to unsigned. The default is
+  /// signed, unless the property is annotated with [@Property(signed: false)].
   static final unsigned = 4;
```

Comment:
Should there be a test for this? There doesn't appear to be any.

Reply:
should be safe to assume it works, since it's completely handled in the core but I've added a test case regardless.